osu.Game/Skinning/SkinConfiguration.cs

```diff
@@ -14,7 +16,39 @@ public class SkinConfiguration : IHasComboColours, IHasCustomColours
     {
         public readonly SkinInfo SkinInfo = new SkinInfo();
 
-        public List<Color4> ComboColours { get; set; } = new List<Color4>();
+        /// <summary>
+        /// Whether to allow <see cref="DefaultComboColours"/> as a fallback list for when no combo colours are provided.
+        /// </summary>
+        internal bool AllowDefaultComboColoursFallback = true;
+
+        public static List<Color4> DefaultComboColours = new List<Color4>
+        {
+            new Color4(255, 192, 0, 255),
+            new Color4(0, 202, 0, 255),
+            new Color4(18, 124, 255, 255),
+            new Color4(242, 24, 57, 255),
+        };
+
+        private List<Color4> comboColours = new List<Color4>();
+
+        public IReadOnlyList<Color4> ComboColours
+        {
+            get
+            {
+                if (comboColours.Count > 0)
+                    return comboColours;
+
+                if (AllowDefaultComboColoursFallback)
+                    return DefaultComboColours;
+
+                return null;
+            }
+            set => comboColours = value.ToList();
+        }
+
+        public void AddComboColours(params Color4[] colours) => colours.ForEach(c => comboColours.Add(c));
```

Comment:
if this is necessary, `AddRange` like before?

Reply:
Right.. 🤦‍♂️